### ISSUE TITLE
Accessibility link is missing from the new front page footer

### ISSUE THREAD

**Issue by tompollard** [MIT-LCP/physionet-build#2463]
There should be an accessibility link at the bottom of the front page footer.

It doesn't appear right now.

Maybe that's because these fields aren't populated [in the .env](https://github.com/MIT-LCP/physionet-build/blob/dev/.env.example):

```
FOOTER_ACCESSIBILITY_PAGE=
FOOTER_ACCESSIBILITY_PAGE_URL=
```

Or maybe the cause is something else.

**Comment by tompollard** [comment]
Fixed in https://github.com/MIT-LCP/physionet-build/pull/2518.
